### DISCUSSION TITLE
new release

### DISCUSSION THREAD

**Discussion by dbil25** [excid3/tailwindcss-stimulus-components#219] [General]
Hey Chris, 
Do you mind releasing a new version of the package with the latest changes (4.0.4)? Thanks in advance, it would be very useful to me!

**Comment by ghost** [comment]
I was looking why the toggle class wasn't working, but that commit is not released yet.
A new release would be nice :-)

**Reply by ghost** [reply]
@dbil25 I just noticed you made that PR. Thanks!!
